data_structures/tower_of_hanoi_problem.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+using namespace std;
+
+void tower_of_hanoi(int n, char rod1, char rod2, char rod3) {
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.